src/app/components/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -17,7 +17,7 @@
   <!-- info area -->
   <div class="list-view-pf-main-info">
     <div class="list-view-pf-left type f8-wi__list-witype">
-      <span almIcon [iconType]="workItem.attributes['system.state']" class="color-grey pull-left"></span>
+      <span  class="pull-left" almIcon [iconType]="workItem.attributes['system.state']"></span>
```

Comment:
extra space before `class`
also, if you're just dropping `color-grey`, then no need to change the attribute ordering of `almIcon` and `class`

Reply:
Got it